packages/components/src/components/flamegraph/FlamegraphMain.vue

```diff
@@ -51,11 +53,6 @@ export default {
       type: Array,
       required: true,
     },
-    focusEvent: {
-      type: Object,
-      default: null,
-      validator: (value) => value === null || value instanceof Event,
-    },
```

Comment:
I'd keep the prop drilling intact and pass this in as far down as from the `v-vs-code-extension` element. I've been slowly pulling out our usage of the store as it makes this kind of thing more difficult to embed in other contexts.

Reply:
As we discussed, I'll keep using the state, but we can re-evaluate later if needed.